x-pack/test/api_integration/apis/endpoint/index_pattern.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect/expect.js';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+
+  describe('Endpoint index pattern API', () => {
```

Comment:
I mentioned this earlier, but what is the purpose of exposing these APIs?

Reply:
Yeah it doesn't have to be, it's only used by the alert list but could be useful if other UIs need to do time based filtering of a list.